js/cool.js

```diff
@@ -25,7 +25,7 @@ function postReady() {
   });
 }
 
-function receiveMessage(hasCloseButton, event) {
+function receiveMessage(close_button_url, event) {
```

Comment:
We could use lowerCamelCase here, for consistency with the rest of the script.\
In the end we should redo all of it.